frontend/app/src/components/nft/NftGalleryItem.vue

```diff
@@ -11,41 +13,145 @@ const props = defineProps({
 });
 
 const { item } = toRefs(props);
+const { addToListImagesDomain, shouldRenderImage } = useNftsStore();
+
+const frontendStore = useFrontendSettingsStore();
+
+const { whitelistedDomainsForNftImages } = storeToRefs(frontendStore);
+const { updateSetting } = frontendStore;
+
+const imageUrlSource: ComputedRef<string | null> = computed(
+  () => get(item).imageUrl
+);
+
+watch(
+  imageUrlSource,
+  image => {
+    if (image) {
+      addToListImagesDomain(image);
```

Comment:
@lukicenturi what I meant with the original comment was to remove the automated domain tracking completely, and instead have either the whitelisting from the gallery item or manually add a domain from the settings.

Also for the settings component the confirmation feels a bit weird.
The chip is added int he component and only when I try to leave do I get the confirmation. Can we change that?

Reply:
Also, we need to do something about videos, this is how they appear
![image](https://user-images.githubusercontent.com/2269732/229590062-200facad-6239-4cf3-9807-625997b7fb93.png)

Reply:
> @lukicenturi what I meant with the original comment was to remove the automated domain tracking completely, and instead have either the whitelisting from the gallery item or manually add a domain from the settings.
> 
> Also for the settings component the confirmation feels a bit weird. The chip is added in the component and only when I try to leave do I get the confirmation. Can we change that?

Okay I get it,
for the confirmation I think onButtonClick will be better than onBlur